src/service/getStepDefinitionLocationLinks.ts

```diff
@@ -14,6 +14,9 @@ export function getStepDefinitionLocationLinks(
   const locationLinks: LocationLink[] = []
   for (const expressionLink of expressionLinks) {
     if (expressionLink.expression.match(stepRange.stepText)) {
+      // the expressionLink.locationLink.targetUri value is not a valid Uri for Windows
+      // It is treated as a directory. We assign a new URL instead. It is valid for Darwin/Linux/Windows
+      expressionLink.locationLink.targetUri = new URL(expressionLink.locationLink.targetUri).href
```

Comment:
Rather than rewriting `targetUri` here, we should create it correctly in the first place. I will create a fix for this.

Reply:
Sure. I wanted to find the root of this value but it was hard to me to find it. As well I don't know if the change could affect other parts of the code.